instance/github.py

```diff
@@ -149,26 +140,48 @@ class PR:
     """
     Representation of a GitHub Pull Request
     """
-    # pylint: disable=too-many-arguments
-    def __init__(self, number, source_fork_name, target_fork_name, branch_name, title, username, body=''):
+
+    def __init__(self, target_fork_name, number):
         self.number = number
-        self.fork_name = source_fork_name
         self.repo_name = target_fork_name
-        self.branch_name = branch_name
-        self.title = title
-        self.username = username
-        self.body = body
+        self.update_data()
+
+    def update_data(self):
+        self._data = get_pr_by_number(target_fork_name, number)
+        self._extra_settings = get_settings_from_pr_body(self.body)
+        self._extra_settings_dict = yaml.load(self.extra_settings) or {}
```

Comment:
Ah, I see - it's true that Google seem to have a strong culture of high performance/speed, at least from what shows from the outside, so I can see how that would become an habit : )

Yup, that works for me to focus on other things - that said, I think it's useful that we took the time to explain our view points, and understand each other's approach and preferences. It will likely show up again in other places in reviews, so that's likely not lost time, even if the changes here are definitely minor.

And sounds good for the next steps - thanks!